default/stringtables/en.txt

```diff
@@ -1725,6 +1725,56 @@ Network port to use for server discovery.
 OPTIONS_DB_NETWORK_MESSAGE_PORT
 Network port to use for client-server messaging.
 
+OPTIONS_DB_UI_DESIGN_PART_FILTER_DESC
+'''DesignWnd Redundant Part Filtering
+
+Deems a first part redundant to a second only if the second is all of:
+
+(i) more powerful
```

Comment:
I aimed to relocate the documentation here, along with minor formatting adjustments.
According to this, both cases fail the first check, so the first part would not be deemed redundant.